golang.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import "fmt" //here is a line coment
```

Comment:
Typo inside comment in token 'coment'. Possible corrections:comment (61),moment (33),cement (17)